client/src/pages/Login.vue

```diff
@@ -0,0 +1,236 @@
+                    <button @click="clearEmailOrUser" class="cancel-email-input-button _1eJKW _16r-S NF_yB tV2C-" tabindex="-1" type="button">
+                        <img src="https://d35aaqx5ub95lt.cloudfront.net/vendor/8857e57af4c38e2caab2df3d086202da.svg">
+                    </button>
+                </div>
+                <div class="text-input-wrapper">
+                    <input class="duolingo-text-input" data-test="email-input" type="password" placeholder="Password">
+                    <router-link class="forgot-button" tabindex="-1" type="button" to="/forgot-password">
+                        FORGOT?
+                    </router-link>
+                </div>
+                <LAButton text="LOG IN" class="w-100" margin="0px"/>
+                <div class="alt-login-divider">
+                    <div class="horizontal-bar"></div>
+                    <div class="upper-case-text-format or-styling">or</div>
+                    <div class="horizontal-bar"></div>
+                </div>
+                <v-row class="w-100" no-gutters>
+                    <v-col class="mr-2">
+                        <LAButton text="FACEBOOK" class="w-100" margin="0px"/>
+                    </v-col>
+                    <v-col class="ml-2">
+                        <LAButton text="GOOGLE" class="w-100" margin="0px"/>
+                    </v-col>
+                </v-row>
+            </v-col>
+          </v-sheet>
+        </v-col>
+      </v-row>
+    </v-container>
+  </template>
+  
+  <script setup>
+  import { ref } from 'vue'
+  import LAButton from '@/components/LAButton.vue'
+  
+  const email = ref('')
+  const password = ref('')
+  const message = ref('')
+  
+
+  const clearEmailOrUser = () =>  {
+    console.log('Before: ' + email.value)
+    email.value = ''
+    console.log('After: ' + email.value)
+  }
+
+  </script>
+  
+  <style scoped>
+    :root {
+        --input-primary-color: rgb(31, 36, 40);    
+        --input-secondary-color: rgb(11, 125, 224); 
+    }
+
+    .button-wrapper .button {
+        margin: 0 !important;
+    }
+
+    .section {
+        height: 100vh;
+        display: flex !important;
+        align-items: start;
+        justify-content: center;
+        text-align: center;
+        font-size: 24px;
+        border: 2px solid red;
+        position: relative;
+    }
+    input {
+        box-shadow: none;   
+    }
+    input, textarea {
+        -webkit-user-select: auto;
+        -moz-user-select: auto;
+        user-select: auto;
+    }
+    .duolingo-text-input {
+        -moz-osx-font-smoothing: grayscale;
+        -webkit-font-smoothing: antialiased;
+        -webkit-appearance: none;
+        -moz-appearance: none;
+        appearance: none;
+        background: none;
+        border: none;
+        box-sizing: border-box;
+        caret-color: rgb(var(--color-macaw));
```

Comment:
are these colors actually being defined anywhere? maybe we either hardcode a filler value for now or setup the styles.scss file to define them.

Reply:
Went through and commented out the css variables. We can go through and re add them where needed but I think the MVP is okay for now. But let me know if you think otherwise.